src/lib/Persistence/Legacy/Content/Handler.php

```diff
@@ -776,6 +777,17 @@ public function addRelation(RelationCreateStruct $createStruct)
         return $relation;
     }
 
+    public function loadRelation(int $relationId): Relation
```

Comment:
CS: missing throws PHPDoc
```suggestion
    /**
     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
     */
    public function loadRelation(int $relationId): Relation
```